src/test/java/duke/ParserTest.java

```diff
@@ -0,0 +1,21 @@
+package duke;
+
+import org.junit.Test;
+
+import static duke.Parser.inputParser;
+import static org.junit.Assert.assertEquals;
+
+public class ParserTest {
+
+    @Test
+    public void emptyTextTest() {
```

Comment:
```suggestion
    public void inputParser_emptyText_arrayOfEmptyStrings() {
```
Perhaps you could follow the java coding standard for naming tests? This applies to other test methods names too